src/go/k8s/controllers/redpanda/cluster_controller.go

```diff
@@ -431,7 +431,7 @@ func (r *ClusterReconciler) createBootstrapStatefulSet(
 								"--memory " + strings.ReplaceAll(memory.String(), "Gi", "G"),
 								"start",
 								"--",
-								"--default-log-level=debug",	// TODO change to info
```

Comment:
sounds like you can squash this line altogether. prev commit added, what the next deletes.

Reply:
Thanks. I just made it "debug" and squashed.